model/team.py

```diff
@@ -17,6 +17,40 @@ def __init__(self, github_team_id, github_team_name, display_name):
         self.team_leads = set()
         self.members = set()
 
+    def get_attachment(self):
+        """Return slack-formatted attachment (dictionary) for team."""
+        text_pairs = [
+            ('Github ID', self.github_team_id),
+            ('Github Team Name', self.github_team_name),
+            ('Display Name', self.display_name),
+            ('Platform', self.platform),
+            ('Team Leads', '\n'.join(self.team_leads)
+                           if self.team_leads != set() else 'n/a'),
+            ('Members', '\n'.join(self.members)
+                        if self.members != set() else 'n/a')
+        ]
+        fields = [{'title': t, 'value': v if v else 'n/a', 'short': True}
+                  for t, v in text_pairs]
+        fallback = str('\n'.join(map(str, text_pairs)))
+
+        return {'fallback': fallback, 'fields': fields}
+
+    def get_basic_attachment(self):
+        """Return basic slack-formatted attachment (dictionary) for team."""
+        text_pairs = [
+            ('Github ID', self.github_team_id),
+            ('Github Team Name', self.github_team_name),
+            ('Display Name', self.display_name),
+            ('Platform', self.platform),
+            ('Team Leads' '\n'.join(self.team_leads)),
+            ('Members', '\n'.join(self.members))
+        ]
+        fields = [{'title': t, 'value': v if v else 'n/a', 'short': True}
+                  for t, v in text_pairs]
```

Comment:
Since the value we are checking is always initialized, we are checking to see if it is an empty string or not. Thus, we can simply remove the tertiary operators in `get_attachment(self)`, and combine the 2 functions into 1 function.

Reply:
will try to see if it works